packages/eslint-plugin/tests/rules/no-unnecessary-type-assertion.test.ts

```diff
@@ -14,7 +14,7 @@ const ruleTester = new RuleTester({
 });
 
 // make sure each test is completely isolated
-// there was some weird behaviour with the mixed ts/tsx test cases without this
+// there was some weird behavior with the mixed ts/tsx test cases without this
```

Comment:
https://grammarist.com/spelling/behavior-behaviour/

Reply:
Yeah, I write Australian english (which is pretty much the same as UK english). Hence I put a `u` in behaviour, colour and favourite.

Which is why I hated setting the spellchecker to american english.. haha